src/components/NotificationsDrawer/NotificationItem.tsx

```diff
@@ -38,7 +38,7 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ notification, onNav
   return (
     <React.Fragment>
       <NotificationDrawerList>
-        <NotificationDrawerListItem variant="info" isRead={notification.read}>
+        <NotificationDrawerListItem aria-label={`Notification item ${notification.id}`} variant="info" isRead={notification.read}>
```

Comment:
aria-label should be an accurate human readable description of an element. Using `id` is not great in this case. I would probably use the title if we want to use this attribute.